scripts/scil_compute_MT_maps.py

```diff
@@ -0,0 +1,376 @@
+proton exchange between non-aqueous protons (from macromolecules and their
+closely associated water molecules, the "bound" pool) and protons in the free
+water pool called aqueous protons. This exchange attenuates the MRI signal,
+introducing microstructure-dependent contrast. MT's effect reflects the
+relative density of macromolecules such as proteins and lipids, it has been
+associated with myelin content in white matter of the brain.
+
+Different contrasts can be done with an off-resonance pulse to saturating the
+protons on non-aqueous molecules a frequency irradiation. The MT maps are
+obtained using three contrasts: single frequency irradiation (MT-on, saturated
+images) and an unsaturated contrast (MT-off); and a T1weighted image as
+reference.
+
+The output consist in two types of images:
+        Three contrasts images : MT-off, MT-on and T1weighted images.
+        MT maps corrected or not for an empiric B1 correction maps.
+
+Input Data recommendation:
+  - it is recommended to use dcm2niix (v1.0.20200331) to convert data
+    https://github.com/rordenlab/dcm2niix/releases/tag/v1.0.20200331
+  - dcm2niix conversion will create all echo files for each contrast and
+    corresponding json files
+  - all input must have a matching json file with the same filename
+  - all contrasts must have a same number of echoes and coregistered
+    between them before running the script.
+  - Mask must be coregistered to the echo images
+  - ANTs can be used for the registration steps (http://stnava.github.io/ANTs/)
+
+
+The output consist in two types of images in two folders :
+  1. Contrasts_MT_maps which contains the 2 contrast images
+      - MT-off.nii.gz : pulses applied at positive frequency
+      - MT-on.nii.gz : pulses applied at negative frequency
+      - T1w.nii.gz : anatomical T1 reference images
+
+
+  2. MT_native_maps which contains the 4 myelin maps
+      - MTR.nii.gz : Magnetization Transfer Ratio map
+      The MT ratio is a measure reflecting the amount of bound protons.
+
+      - MTsat.nii.gz : Magnetization Transfer saturation map
+      The MT saturation is a pseudo-quantitative maps representing
+      the signal change between the bound and free water pools.
+
+>>> scil_compute_ihMT_maps.py path/to/output/directory path/to/mask_bin.nii.gz
+    --in_mtoff path/to/echo*mtoff.nii.gz --in_mton path/to/echo*mton.nii.gz
+    --in_t1w path/to/echo*T1w.nii.gz
+
+"""
+
+import argparse
+import os
+import json
+
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+
+EPILOG = """
+Helms G, Dathe H, Kallenberg K, Dechent P. High-resolution maps of
+magnetization transfer with inherent correction for RF inhomogeneity
+and T1 relaxation obtained from 3D FLASH MRI. Magnetic Resonance in Medicine.
+2008;60(6):1396‑407.
```

Comment:
@Manonedde could you please remove the - in `1396‑407` and readd it. You copy paste directly the URL and this character is not the real `-`.